data/maps/hide_show_data.asm

```diff
@@ -263,306 +263,306 @@ MissableObjects:
+	db ROCKET_HIDEOUT_B4F, ROCKETHIDEOUTB4F_HP_UP,         SHOW
+	db ROCKET_HIDEOUT_B4F, ROCKETHIDEOUTB4F_TM_RAZOR_WIND, SHOW
+	db ROCKET_HIDEOUT_B4F, ROCKETHIDEOUTB4F_IRON,          SHOW
+	db ROCKET_HIDEOUT_B4F, ROCKETHIDEOUTB4F_SILPH_SCOPE,   HIDE
+	db ROCKET_HIDEOUT_B4F, ROCKETHIDEOUTB4F_LIFT_KEY,      HIDE
 SilphCo2FHS:
-	db SILPH_CO_2F, $01, SHOW
-	db SILPH_CO_2F, $02, SHOW
-	db SILPH_CO_2F, $03, SHOW
-	db SILPH_CO_2F, $04, SHOW
-	db SILPH_CO_2F, $05, SHOW
+	db SILPH_CO_2F, SILPHCO2F_SILPH_WORKER_F, SHOW
+	db SILPH_CO_2F, SILPHCO2F_SCIENTIST1,     SHOW
+	db SILPH_CO_2F, SILPHCO2F_SCIENTIST2,     SHOW
+	db SILPH_CO_2F, SILPHCO2F_ROCKET1,        SHOW
+	db SILPH_CO_2F, SILPHCO2F_ROCKET2,        SHOW
 SilphCo3FHS:
-	db SILPH_CO_3F, $02, SHOW
-	db SILPH_CO_3F, $03, SHOW
-	db SILPH_CO_3F, $04, SHOW
+	db SILPH_CO_3F, SILPHCO3F_ROCKET,       SHOW
+	db SILPH_CO_3F, SILPHCO3F_SCIENTIST,    SHOW
+	db SILPH_CO_3F, SILPHCO3F_HYPER_POTION, SHOW
 SilphCo4FHS:
-	db SILPH_CO_4F, $02, SHOW
-	db SILPH_CO_4F, $03, SHOW
-	db SILPH_CO_4F, $04, SHOW
-	db SILPH_CO_4F, $05, SHOW
-	db SILPH_CO_4F, $06, SHOW
-	db SILPH_CO_4F, $07, SHOW
+	db SILPH_CO_4F, SILPHCO4F_ROCKET1,     SHOW
+	db SILPH_CO_4F, SILPHCO4F_SCIENTIST,   SHOW
+	db SILPH_CO_4F, SILPHCO4F_ROCKET2,     SHOW
+	db SILPH_CO_4F, SILPHCO4F_FULL_HEAL,   SHOW
+	db SILPH_CO_4F, SILPHCO4F_MAX_REVIVE,  SHOW
+	db SILPH_CO_4F, SILPHCO4F_ESCAPE_ROPE, SHOW
 SilphCo5FHS:
-	db SILPH_CO_5F, $02, SHOW
-	db SILPH_CO_5F, $03, SHOW
-	db SILPH_CO_5F, $04, SHOW
-	db SILPH_CO_5F, $05, SHOW
-	db SILPH_CO_5F, $06, SHOW
-	db SILPH_CO_5F, $07, SHOW
-	db SILPH_CO_5F, $08, SHOW
+	db SILPH_CO_5F, SILPHCO5F_ROCKET1,      SHOW
+	db SILPH_CO_5F, SILPHCO5F_SCIENTIST,    SHOW
+	db SILPH_CO_5F, SILPHCO5F_ROCKER,       SHOW
+	db SILPH_CO_5F, SILPHCO5F_ROCKET2,      SHOW
+	db SILPH_CO_5F, SILPHCO5F_TM_TAKE_DOWN, SHOW
+	db SILPH_CO_5F, SILPHCO5F_PROTEIN,      SHOW
+	db SILPH_CO_5F, SILPHCO5F_CARD_KEY,     SHOW
 SilphCo6FHS:
-	db SILPH_CO_6F, $06, SHOW
-	db SILPH_CO_6F, $07, SHOW
-	db SILPH_CO_6F, $08, SHOW
-	db SILPH_CO_6F, $09, SHOW
-	db SILPH_CO_6F, $0A, SHOW
+	db SILPH_CO_6F, SILPHCO6F_ROCKET1,    SHOW
+	db SILPH_CO_6F, SILPHCO6F_SCIENTIST,  SHOW
+	db SILPH_CO_6F, SILPHCO6F_ROCKET2,    SHOW
+	db SILPH_CO_6F, SILPHCO6F_HP_UP,      SHOW
+	db SILPH_CO_6F, SILPHCO6F_X_ACCURACY, SHOW
 SilphCo7FHS:
-	db SILPH_CO_7F, $05, SHOW
-	db SILPH_CO_7F, $06, SHOW
-	db SILPH_CO_7F, $07, SHOW
-	db SILPH_CO_7F, $08, SHOW
-	db SILPH_CO_7F, $09, SHOW
-	db SILPH_CO_7F, $0A, SHOW
-	db SILPH_CO_7F, $0B, SHOW
-	db SILPH_CO_7F, $0C, SHOW
+	db SILPH_CO_7F, SILPHCO7F_ROCKET1,         SHOW
+	db SILPH_CO_7F, SILPHCO7F_SCIENTIST,       SHOW
+	db SILPH_CO_7F, SILPHCO7F_ROCKET2,         SHOW
+	db SILPH_CO_7F, SILPHCO7F_ROCKET3,         SHOW
+	db SILPH_CO_7F, SILPHCO7F_RIVAL,           SHOW
+	db SILPH_CO_7F, SILPHCO7F_CALCIUM,         SHOW
+	db SILPH_CO_7F, SILPHCO7F_TM_SWORDS_DANCE, SHOW
+	db SILPH_CO_7F, $0C,                       SHOW ; unused
```

Comment:
I'd say this should have the comment `; unreferenced` instead, since it is never referenced anywhere. `; unused` should be stuff that is referenced, but not used.

Minor nitpick. lol

Reply:
I would instead put a `const_export SILPHCO7F_UNUSED ; no corresponding object_event` in data/maps/objects/SilphCo7F.asm. That way someone editing Silph Co 7F won't accidentally make a $Cth event which happens to correspond to this hide/show.